src/expressionDescriptor.ts

```diff
@@ -461,7 +461,20 @@ export class ExpressionDescriptor {
       description = allDescription;
     } else if (!StringUtilities.containsAny(expression, ["/", "-", ","])) {
       description = StringUtilities.format(getDescriptionFormat(expression), getSingleItemDescription(expression));
-    } else if (expression.indexOf("/") > -1) {
+    } else if (expression.indexOf("/") > -1 && expression.indexOf(",") > -1) {
```

Comment:
The if block on 492-494 (`if (betweenSegmentDescription.indexOf(", ")...`) can probably be removed now since it shouldn't be hit with this change.